bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -4,116 +4,122 @@ CE02SHBP-MJ01C-05-IP1,,Sensor,0,JBox Port J05 (LJ01D),JBox Port J05 (LJ01D),,,,,
+CGCON-BDCL11-50004,OL000263,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0004,,,,CP03ISSM-00004-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50005,OL000231,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0005,,,,CP01CNSM-00005-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50006,OL000524,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0006,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-50007,OL000324,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0007,,,,CE02SHSM-00003-DCL11,Oregon State University
+CGCON-BDCL11-50008,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0008,,,,CE09OSSM-00005-SBD11,Oregon State University
+CGCON-BDCL11-50009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2015-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60002,OL000594,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2016-0002,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-60701,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,60701,,,,DELET?,
+CGCON-BDCL11-70001,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0001,,,,CP04OSSM-00006-DCL11,Woods Hole Oceanographic Institution
+CGCON-BDCL11-70009,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,2017-0009,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL11-99999,,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,UNKNOWN,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00001,OL000487,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,001,,,,CP01CNSM-00001-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00002,OL000438,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3703-00235-0002,,,,CE07SHSM-00003-DCL12,Oregon State University
+CGCON-BDCL12-00004,OL000501,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,004,,,,CP01CNSM-00002-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00006,OL000382 OL000534 OL000558,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,006,,,,CP04OSSM-00003-DCL12,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00007,OL000546,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,007,,,,,Woods Hole Oceanographic Institution
+CGCON-BDCL12-00008,OL000393 OL000570 R00019,Sensor,0,Surface Buoy Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,008,,,,CE04OSSM-00002-SBD12,Oregon State University; Woods Hole Oceanographic Institution
```

Comment:
eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet shows 2nd entry for mooring info: CP03ISSM-00001-DCL12